src/components/DateTimePicker/DateTimePicker.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import Picker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+const DateTimePicker = props => {
+  const { date, setDate, customInput, customInputProps, isTimeOnly, isSmallScreen, ...rest } = props;
+
+  const fixOnEnter = e => {
+    // makes sure input loses focus on enter, to avoid date parsing bugs
+    if (e.key.toLowerCase() === "enter") {
+      setTimeout(() => document.activeElement.blur());
```

Comment:
leaving the focus inside the input after the date has been parsed leads to some unpredictable behaviour: say the user writes 1.1.19, hits enter, this turns into 01/01/2019, the cursor is still there, so the user hits backspace and writes another date, 2.1.19. now as soon as he reaches 9, without even pressing enter, the date becomes 02/01/2019. but what if the user wanted to write 1999 this time? after blurring the parser treats user input in a more predictable manner, and it doesnt break keyboard navigation (user can still hit TAB to go to the next field). I tweaked it so that if we define another onKeyDown form props, it will be called after this fix.

Reply:
I see, sounds to be pretty creepy :) I think we can continue with this for now.